### PR TITLE
[Hutch] Fixes output format of Hutch to match open-spiel expectation.

### PR DESCRIPTION
**Pull request by karthik-ramanathan-3006**
**What's the problem?**  
OpenSpiel is fed by JSON output from Explain which is like so:
```
[
  {
    "Plan": {
      "Node Type": "Sort",
      "Parallel Aware": false,
      "Async Capable": false,
      "Startup Cost": 1047.14,
      "Total Cost": 1070.73,
      "Plan Rows": 9435,
      "Plan Width": 62,
      "Sort Key": ["d.d_city"],
      "Plans": [
        {
          "Node Type": "Nested Loop",
          "Parent Relationship": "Outer",
          "Parallel Aware": false,
          "Async Capable": false,
          "Join Type": "Inner",
          "Startup Cost": 3.23,
          "Total Cost": 424.25,
          "Plan Rows": 9435,
          "Plan Width": 62,
          "Inner Unique": false,
          "Join Filter": "(o.no_w_id = w.w_id)",
          <snip>
        }
       ]
    }
  }
]  
```

OpenSpiel looks for a "Plan" key in the top level dictionary, and traverses the plan tree through the "Plans" array in each node.
This PR modifies Hutch to spit out a similar structure which now looks as follows:
```
[
  {
    "Tscout": {
      "node": "node-0",
      "tag": "ExecSort",
      "Sort_plan_type": 42,
      "Sort_plan_startup_cost": 1047.143336,
      "Sort_plan_total_cost": 1070.730836,
      "Sort_plan_plan_rows": 9435.000000,
      "Sort_plan_plan_width": 62,
      "Sort_plan_parallel_aware": false,
      "Sort_plan_parallel_safe": true,
      "Sort_plan_async_capable": false,
      "Sort_plan_plan_node_id": 0,
      "Sort_plan_targetlist": "<skipped>",
      "Sort_plan_qual": "<skipped>",
      "Sort_plan_lefttree": "<skipped>",
      "Sort_plan_righttree": "<skipped>",
      "Sort_plan_initPlan": "<skipped>",
      "Sort_plan_extParam": "<skipped>",
      "Sort_plan_allParam": "<skipped>",
      "Sort_numCols": 1,
      "Sort_sortColIdx": "<skipped>",
      "Sort_sortOperators": "<skipped>",
      "Sort_collations": "<skipped>",
      "Sort_nullsFirst": "<skipped>",
      "Plans": [
        {
          "node": "node-1",
          "tag": "ExecNestLoop",
          "NestLoop_join_plan_type": 37,
          "NestLoop_join_plan_startup_cost": 3.225000,
          "NestLoop_join_plan_total_cost": 424.253750,
          "NestLoop_join_plan_plan_rows": 9435.000000,
          "NestLoop_join_plan_plan_width": 62,
          "NestLoop_join_plan_parallel_aware": false,
          "NestLoop_join_plan_parallel_safe": true,
          "NestLoop_join_plan_async_capable": false,
          "NestLoop_join_plan_plan_node_id": 1,
          "NestLoop_join_plan_targetlist": "<skipped>",
          "NestLoop_join_plan_qual": "<skipped>",
          "NestLoop_join_plan_lefttree": "<skipped>",
          "NestLoop_join_plan_righttree": "<skipped>",
          "NestLoop_join_plan_initPlan": "<skipped>",
          "NestLoop_join_plan_extParam": "<skipped>",
          "NestLoop_join_plan_allParam": "<skipped>",
          "NestLoop_join_jointype": 0,
          "NestLoop_join_inner_unique": false,
          "NestLoop_join_joinqual": "<skipped>",
          "NestLoop_nestParams": "<skipped>",
          "Plans": [
            {},
            {},
          ]
```

### CODE REVIEW

File: cmudb/extensions/hutch/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Set up the TScout extension.
```
Comment (17zhangw) [inline cmudb/extensions/hutch/CMakeLists.txt]: I'm under the impression that we are no longer using cmake?
If so, do we still need these files?

(@lmwnshn)

**Review by 17zhangw — CHANGES_REQUESTED** [review]
Changes in general seem fine to me.

A clarification note: does the actual `EXPLAIN` output show `<skipped>` as in the PR description? Is there a reason for this? How simple is it to insert a dummy/sentinel value instead (our of curiosity)?

File: cmudb/extensions/hutch/CMakeLists.txt (hunk 1 above)
Comment (karthik-ramanathan-3006) [reply cmudb/extensions/hutch/CMakeLists.txt]: No idea what happened here. 
I re-merged the upstream cmu-db/postgres and this sorted itself out.

**Comment by karthik-ramanathan-3006** [comment]
> Changes in general seem fine to me.
> 
> A clarification note: does the actual `EXPLAIN` output show `<skipped>` as in the PR description? Is there a reason for this? How simple is it to insert a dummy/sentinel value instead (our of curiosity)?

@17zhangw - Yes, the `EXPLAIN ( format tscout )` output actually shows `<skipped>`. The reason is that the values of these keys are not scalars but pointers to other structs. It shouldn't be too hard to do some pointer-chasing in order to at-least extract the top-level values in these structs. The change would mostly be isolated to `tscout_feature_gen.py` in hutch, which would have to code-gen information about these structs. I excluded doing this for the first cut, purely for time reasons. Let me know your thoughts on this!

**Review by 17zhangw — APPROVED** [review]

**Pull request merged**